sdk/go/pulumi/context_test.go

```diff
@@ -494,10 +494,10 @@ func TestRegisterResource_aliasesSpecs(t *testing.T) {
 			// alias specs.
 			// So if that's needed, wrap the monitor to claim it
 			// does.
-			if tt.supportsAliasSpecs {
+			if !tt.supportsAliasSpecs {
```

Comment:
The changes necessary in this test made me a little bit worried that we'd be breaking some backwards compatibility here, but we're testing internal things here, so maybe it's okay?  I'd appreciate some extra eyes here.

Reply:
Yeh this looks fine